src/components/radioButton/RadioButton.tsx

```diff
@@ -140,21 +107,8 @@ class RadioButton extends BaseComponent {
     }
   };
 
-  getAccessibilityProps = () => {
-    const {label = '', selected, disabled} = this.getThemeProps();
-    const selectedAccessibilityText = selected ? 'selected' : 'unselected';
-    const accessibilityLabel = `${selectedAccessibilityText}. ${label}`;
-
-    return {
-      accessible: true,
-      accessibilityStates: disabled ? ['disabled'] : undefined,
```

Comment:
I was unable to have this method with accessibilityStates & accessibilityRole - it had errors of unrecognised types that I did not find a solution to.
I've created `getSomeAccessibilityProps` a bit lower in the code to at least avoid too much code in the render.

Reply:
I prefer to have `//@ts-ignore` above it instead of changing the code

Reply:
Reverted